client/state/selectors/is-iframe-for-html-element.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import getSelectedSiteId from 'state/ui/selectors/get-selected-site-id';
+import getSectionName from 'state/ui/selectors/get-section-name';
+
+const isIframeForHtmlElement = state =>
+	!! ( getSelectedSiteId( state ) && getSectionName( state ) === 'gutenberg-editor' );
```

Comment:
Would it make sense to include `customizer` here in addition to `gutenberg-editor` for consistency?

Reply:
@vindl happy to...  Where is that set presently?

Reply:
I'm seeing these:
* https://github.com/Automattic/wp-calypso/blob/dc7ee470968f9137e19fb3a57f7a8c86d7fe4b05/client/gutenberg/editor/calypsoify-iframe.tsx#L595
* https://github.com/Automattic/wp-calypso/blob/b49fc5f9c943b3b30987ed135a964d40146be587/client/my-sites/customize/main.jsx#L336

...but they're setting the `is-iframe` class on the "main" `<div>` -- not the `<html>` node, so, unless I'm missing something, it's semantically different enough, I'd like to not conflate them.

Reply:
It works in the same way so I was expecting to see it in the document layout, however it's only present here in the Customizer case: 🤔 

https://github.com/Automattic/wp-calypso/blob/b49fc5f9c943b3b30987ed135a964d40146be587/client/my-sites/customize/main.jsx#L336

In any case, it's not something I'd block this fix on.